build.gradle

```diff
@@ -32,6 +32,7 @@ allprojects {
   targetCompatibility = 11
 
   def idamBomVersion = '2.2.6'
+  ext['idam.api-spec.version'] = '2.17.8-RC4'
```

Comment:
Just to confirm, is this temporary and will be replaced by a new version of the boom?

Reply:
@radoslaw-orlowski-amido yes that's right. I'm going to get rid of that with my next commit.